app/src/main/java/me/hufman/androidautoidrive/phoneui/MusicQueuePageFragment.kt

```diff
@@ -0,0 +1,165 @@
+package me.hufman.androidautoidrive.phoneui
+
+import android.app.Activity
+import android.arch.lifecycle.ViewModelProviders
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.drawable.Animatable2
+import android.graphics.drawable.AnimatedVectorDrawable
+import android.graphics.drawable.Drawable
+import android.os.Bundle
+import android.os.Handler
+import android.support.v4.app.Fragment
+import android.support.v7.widget.LinearLayoutManager
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import kotlinx.android.synthetic.main.music_queuepage.*
+import kotlinx.coroutines.*
+import me.hufman.androidautoidrive.R
+import me.hufman.androidautoidrive.music.MusicController
+import me.hufman.androidautoidrive.music.MusicMetadata
+import me.hufman.androidautoidrive.music.QueueMetadata
+import kotlin.collections.ArrayList
+import kotlin.coroutines.CoroutineContext
+
+class MusicQueuePageFragment: Fragment(), CoroutineScope {
+	override val coroutineContext: CoroutineContext
+		get() = Dispatchers.Main
```

Comment:
I think this is no longer used and can be cleaned up, along with the related imports

Reply:
Yeah you're correct, I removed all the coroutine related stuff from this file